### PR TITLE
add openshit profile.

### PR DESCRIPTION
**Pull request by morvencao**
Please provide a description for what this PR is for.
Add openshift profile for https://github.com/istio/istio/pull/28076

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ X ] Docs
[ X ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/docs/setup/additional-setup/config-profiles/index.md [hunk 1]
```diff
@@ -42,19 +42,22 @@ for your specific needs. The following built-in configuration profiles are curre
 1. **preview**: the preview profile contains features that are experimental. This is intended to explore new features
                 coming to Istio. Stability, security, and performance are not guaranteed - use at your own risk.
 
+1. **openshift**: the openshift profile is provided as a starting point of installing Istio on an OpenShift platform.
```
Comment (shamsher31) [inline content/en/docs/setup/additional-setup/config-profiles/index.md:45]: @frankbu suggested https://github.com/istio/istio.io/pull/8434#discussion_r516724548

File: content/en/docs/setup/additional-setup/config-profiles/index.md [hunk 2]
```diff
@@ -42,19 +42,22 @@ for your specific needs. The following built-in configuration profiles are curre
...
 {{< tip >}}
 Some additional vendor-specific configuration profiles are also available.
 For more information, refer to the [setup instructions](/docs/setup/platform-setup) for your platform.
 {{< /tip >}}
 
 The components marked as **X** are installed within each profile:
 
-|     | default | demo | minimal | remote | empty | preview |
-| --- | --- | --- | --- | --- | --- | --- |
+|     | default | demo | minimal | remote | empty | preview | openshift |
```
Comment (frankbu) [inline content/en/docs/setup/additional-setup/config-profiles/index.md:54]: If you want to describe what components are included in the openshift profile, it should be added in the OpenShift platform setup doc, not here.

File: content/en/docs/setup/additional-setup/config-profiles/index.md (hunk 1 above)
Comment (morvencao) [reply content/en/docs/setup/additional-setup/config-profiles/index.md:45]: thanks!

**Comment by morvencao** [comment]
Thanks, closing this...

**Pull request closed without merging**